build.mk

```diff
@@ -21,6 +21,13 @@ else
 	COPY = cp -rf
 endif
 
+MKDIR:=
+ifeq ($(OS_NAME),Windows)
+	MKDIR = mkdir
+else
+	MKDIR = mkdir -p
+endif
```

Comment:
It looks like you are not using this define.

Reply:
Whoops! Yes, it looks like the command got removed when I merged from upstream.. not needed anymore